install/update_085_0853.php

```diff
@@ -88,23 +88,29 @@ function update085to0853() {
                 ) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci";
       $DB->queryOrDie($query, "0.85 add table glpi_items_tickets");
 
-      $query = "SELECT `itemtype`, `items_id`, `id`
-                FROM `glpi_tickets`
-                WHERE `itemtype` IS NOT NULL
-                    AND `itemtype` <> ''
-                AND `items_id` != 0";
-
-      if ($result = $DB->query($query)) {
-         if ($DB->numrows($result)>0) {
-            while ($data = $DB->fetch_assoc($result)) {
-                $query = "INSERT INTO `glpi_items_tickets`
-                             (`id`, `items_id`, `itemtype`, `tickets_id`)
-                          VALUES (NULL, '".$data['items_id']."', '".$data['itemtype']."', '".$data['id']."')";
-                $DB->queryOrDie($query, "0.85 associated ticket sitems migration");
-            }
-
+      $ticketsIterator = $DB->request([
+         'SELECT' => ["itemtype", "items_id", "id"],
+         'FROM'   => "glpi_tickets",
+         'WHERE'  => [
+            "itemtype" => new \QueryExpression("IS NOT NULL"),
```

Comment:
```suggestion
             'NOT' => ['itemtype' => null],
```
This will also prevent next line to erase current one (they use same key).